### PR TITLE
fix: update color warning [TOL-1801]

### PR DESCRIPTION
**Pull request by 2wce**
# Purpose of PR

In the `EntityStatusBadge` component when we have a scheduled action the icon fill color (orange-400 is the value for the warning variant for icons) is different from the status text color (orange 600 is the default). This was not intentional so this PR aims to fix that

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Comment by cf-remylenoir** [comment]
_The failing CI is not related to your changes, waiting on the PR fix to be green._

File: package-lock.json
Comment (cf-remylenoir) [inline package-lock.json:1]: This can be removed 👀

**Comment by 2wce** [comment]
Cool I'll update the PR on Monday so @cf-remylenoir we can merge then

**Review by Lelith — APPROVED** [review]
@2wce don 't forget to run npx changeset to get the version bump automated after merge

**Review by cf-remylenoir — CHANGES_REQUESTED** [review]
Happy to approve when the lock is removed + changeset added 🙂

**Review by cf-remylenoir — APPROVED** [review]
Apologies for the late review, I had my head down with the Studio GA 😄 Thank you for your contribution!

**Comment by cf-remylenoir** [comment]
@all-contributors please add @2wce for code

**Pull request merged**
